docs/Operators.md

```diff
@@ -5058,6 +5059,50 @@ expect(node, inputs=[x, y], outputs=[z],
 </details>
 
 
+### <a name="Multinomial"></a><a name="multinomial">**Multinomial**</a>
+
+  Generate a tensor of samples from a multinomial distribution according to the probabilities
+  of each of the possible outcomes.
+
+#### Version
+
+This version of the operator has been available since version 1 of the default ONNX operator set.
```

Comment:
this is a new operator, It has not been available since op_set version 1.

Reply:
@linkerzhang produces this. I think simply by running gen_doc. Ke, can you look into this feedback? I somehow had problem generating docs in my machine with gen_doc.py. It decimated the doc as a result but without an error.

Reply:
good catch. fixed.